### ISSUE TITLE
DSB Item - Non-Bank Lending Candidate Standards

### ISSUE THREAD

**Issue by markverstege** [ConsumerDataStandardsAustralia/future-plan#78]
# Problem Statement

A series of consultations for non-bank lending are planned to be opened and progressively developed over this quarter with a view to completing the Non-Bank Lending standards to candidate level by early 2023.

**Comment by nils-work** [comment]
Treasury material for reference -
- [Consumer Data Right - Strategic Assessment Outcomes](https://treasury.gov.au/publication/p2022-242997) (24 January 2022)
- [Consumer Data Right Sectoral Assessment for Non-Bank Lending – Open Finance](https://treasury.gov.au/consultation/c2022-253782) (15 March 2022)
- [Consumer Data Right – Sectoral Assessment for Non-Bank Lending – Final Report](https://treasury.gov.au/publication/p2022-300402) (19 August 2022)
- [Consumer Data Right – Non-Bank Lending draft designation instrument](https://treasury.gov.au/consultation/c2022-300402) (19 August 2022)
- [Consumer Data Right – Open Banking sectoral assessment and draft designation instrument – minor and technical amendment](https://treasury.gov.au/consultation/c2022-300402-ob) (19 August 2022)

**Comment by nils-work** [comment]
[Noting Paper 292 - Approach to developing Data Standards for the Non-Bank Lending Sector](https://github.com/ConsumerDataStandardsAustralia/standards/issues/292) has been published.

**Comment by nils-work** [comment]
The next stage for Non-Bank Lending Standards will be a series of Decision Proposals covering Draft Standards.

**Comment by nils-work** [comment]
The following papers have been posted - 
- [Decision Proposal 316 - Non-Bank Lending sector alignment](https://github.com/ConsumerDataStandardsAustralia/standards/issues/316)
- [Decision Proposal 320 - CX Standards | Non-Bank Lending](https://github.com/ConsumerDataStandardsAustralia/standards/issues/320)

**Comment by nils-work** [comment]
Exposure draft rules to expand the CDR to the non‑bank lenders sector have been published -
- [Consumer Data Right rules – expansion to the non-bank lending sector](https://treasury.gov.au/consultation/c2023-434434-expansion)

**Comment by nils-work** [comment]
The following paper has been published to consider the introduction of Buy Now, Pay Later products to the Banking and Non-Bank Lenders sector - 
- [#317 - Decision Proposal 317 - ‘Buy Now, Pay Later’ Product and Account Detail](https://github.com/ConsumerDataStandardsAustralia/standards/issues/317)

**Comment by nils-work** [comment]
Resources that may be valuable for participants in the Non-Bank Lending sector - 
- [CDR information map](https://www.cdr.gov.au/for-providers/cdr-information-map)
- [White label products](https://www.cdr.gov.au/resources/guides#goto-white-label-products)
- [Compliance guide for data holders - banking sector](https://www.cdr.gov.au/resources/guides/compliance-guide-data-holders-banking-sector)

**Comment by nils-work** [comment]
Decision Proposal 318 has been published to support a holistic consultation on the [Non-Bank Lending Draft Standards](https://consumerdatastandardsaustralia.github.io/standards/includes/additional/drafts/non-bank-lending/banking-non-bank-lending.html), including Consumer Experience Data Language and Technical Standards.

The proposal is attached to the following comment: https://github.com/ConsumerDataStandardsAustralia/standards/issues/318#issuecomment-1807753374

**Comment by nils-work** [comment]
Non-Bank Lending Standards were made as [Candidate Standards](https://consumerdatastandardsaustralia.github.io/standards/includes/additional/candidates/non-bank-lending/banking-non-bank-lending.html) in Standards version 1.29.0.

**Comment by nils-work** [comment]
Consultation is now open for [Consumer Data Right Rules – non-bank lending and banking data scope](https://treasury.gov.au/consultation/c2024-598346).

**Comment by nils-work** [comment]
Rules regarding the extension of the consumer data right to the non-bank lenders sector have been made.
See the [amendment rules](https://www.legislation.gov.au/F2025L00272/latest/text) and [explanatory statement](https://www.legislation.gov.au/F2025L00272/latest/text/explanatory-statement).

**Comment by nils-work** [comment]
[#367 - Consultation Draft 367 - March 2025 Rules - Draft Standards](https://github.com/ConsumerDataStandardsAustralia/standards/issues/367) is now open for feedback.
